python/python/glide/async_commands/core.py

```diff
@@ -5428,3 +5428,152 @@ def try_get_pubsub_message(self) -> Optional[PubSubMsg]:
             >>> pubsub_msg = listening_client.try_get_pubsub_message()
         """
         ...
+
+    async def lcs(
+        self,
+        key1: str,
+        key2: str,
+    ) -> str:
+        """
+        Returns the longest common subsequence between strings stored at key1 and key2.
+        See https://valkey.io/commands/lcs for more details.
+
+        Args:
+            key1 (str): The key that stores the first string.
+            key2 (str): The key that stores the second string.
+
+        Returns:
+            A String containing the longest common subsequence between the 2 strings.
+            An empty String is returned if the keys do not exist or have no common subsequences.
+
+        Examples:
+            >>> await client.mset({"testKey1" : "abcd", "testKey2": "axcd"})
+                'OK'
+            >>> await client.lcs("testKey1", "testKey2")
+                'acd'
+
+        Since: Redis version 7.0.0.
+        """
+        args = [key1, key2]
+
+        return cast(
+            str,
+            await self._execute_command(RequestType.LCS, args),
+        )
+
+    async def lcs_len(
+        self,
+        key1: str,
+        key2: str,
+    ) -> int:
+        """
+        Returns the length of the longest common subsequence between strings stored at key1 and key2.
+        See https://valkey.io/commands/lcs for more details.
+
+        Args:
+            key1 (str): The key that stores the first string.
+            key2 (str): The key that stores the second string.
+
+        Returns:
+            The length of the longest common subsequence between the 2 strings.
+
+        Examples:
+            >>> await client.mset({"testKey1" : "abcd", "testKey2": "axcd"})
+                'OK'
+            >>> await client.lcs_len("testKey1", "testKey2")
+                3  # the length of the longest common subsequence between these 2 strings is 2.
```

Comment:
Should it be?
```suggestion
                3  # the length of the longest common subsequence between these 2 strings is 3.
```

Reply:
The longest sequence between 'abcd' and 'axcd' should be 2 (the 'cd' sequence), so we should return 2:
```suggestion
                2  # the length of the longest common subsequence between these 2 strings is 2 (the "cd" sequence).
```

Reply:
Like I mentioned in the other reply, `Longest common subsequence` and `Longest common substring` are defined differently.